src/csv_overrides.py

```
@@ -136,16 +136,15 @@ def update_dicts(
                     "value": value,
                     "list": default_list_name,
                 }
-            else:
-                raise
```

Should we at least log something to terminal here?  I don't think it's great to just fail silently

---

It doesn't fail; it just skips that one. But we can add a comment

---

done